app/docs/0.11.x/configuration.md

```diff
@@ -201,19 +206,45 @@ http {
 
 [Back to TOC](#table-of-contents)
 
-#### Embed Kong in OpenResty
+#### Starting Kong via Nginx
 
-If you are running your own OpenResty servers, you can also easily embed Kong
-by including the Kong Nginx sub-configuration using the `include` directive
+The command `kong start` launches Kong, internally initiating an Nginx
+process. If you are running your own OpenResty servers, you may want
+to flip this process over, starting Nginx yourself and having its
+configuration launch Kong. This may also desirable in containerized
+environments, to ensure that the Nginx master process is the foreground
+process in the container, instead of the `kong` CLI command.
+
+To launch Kong from within Nginx, you need an Nginx configuration file
+which includes the Kong Nginx sub-configuration using the `include` directive
 (similar to the examples of the previous section). However, you will need the
-final configuration and not the template. For this, use the `compile` command,
-which outputs a fully-compiled Nginx sub-configuration to `stdout`:
+final configuration and not the template. To prepare this configuration file,
+use the `kong prepare` command, which generates `nginx.conf` and
```

Comment:
Added an extra bit mentioning logs and ssl, and a summary at the end of the section with a tl;dr about the relationship of the various files.